spec/plans/results.fmf

```diff
@@ -110,6 +110,12 @@ description: |
     fail
         Test execution successfully finished and failed.
 
+    skip
+        Test was executed but the implementation determined
```

Comment:
Hm, sounds a bit contradictory `was executed` versus `it would not be run`. Perhaps, `Test was discovered...`?